client/src/component/InputDate/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import 'date-fns';
+import DateFnsUtils from '@date-io/date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+
+const InputDate = ({ label }) => {
+  const [selectedDate, setSelectedDate] = React.useState(new Date());
+
+  const handleDateChange = (date) => {
+    setSelectedDate(date);
+  };
+
+  return (
+    <MuiPickersUtilsProvider utils={DateFnsUtils}>
+      <KeyboardDatePicker
+        disableToolbar
+        variant="inline"
+        format="MM/dd/yyyy"
```

Comment:
please change the format to match the format we use in our fake data.

format in frontEnd
![image](https://user-images.githubusercontent.com/32321796/113628167-2c135c00-966d-11eb-9b49-714668033069.png)

format in dataBase
![image](https://user-images.githubusercontent.com/32321796/113628242-44837680-966d-11eb-822c-ef927b9c0cec.png)